addons/mail/static/src/js/followers.js

```diff
@@ -315,7 +315,7 @@ var Followers = AbstractField.extend({
         var self = this;
```

Comment:
i know it's not your job, but maybe just add minimal documentation on this function (because you change the signature)